src/config/routes/2.1.php

```diff
@@ -470,6 +470,16 @@
                 'PUT',
             ],
     ],
+
+    [
+        'path'       => '/users/[0-9]+/talk_comments',
+        'controller' => UsersController::class,
+        'action'     => 'deleteTalkComments',
+        'verbs'      =>
+            [
```

Comment:
Do we want to be consistent and continue to use the weird wrapping, or perhaps adjust it?

```diff
diff --git a/src/config/routes/2.1.php b/src/config/routes/2.1.php
index 425e142..c41ac7f 100644
--- a/src/config/routes/2.1.php
+++ b/src/config/routes/2.1.php
@@ -42,10 +42,9 @@ return [
         'path'       => '/events/[\\d]+/hosts/[\\d]+',
         'controller' => EventHostsController::class,
         'action'     => 'removeHostFromEvent',
-        'verbs'      =>
-            [
-                'DELETE',
-            ],
+        'verbs'      => [
+            'DELETE',
+        ],
     ],
     [
         'path'       => '/events(/[^/]+)*/comments/reported',
```

Reply:
Hm, not sure, do we have a standard for this? Also not sure if this is picked up by the cs fixer but will check.

Reply:
Currently we do not have anything configured for `phpcs`, so please ignore my comment.